source/model/Device.py

```diff
@@ -0,0 +1,34 @@
+from abc import ABC, abstractmethod
+
+
+class Device(ABC):
+    cntr = 0
```

Comment:
Can the id not be set when we store it in the database? That way the class is not responsible for its own id counter